.github/actions/inductor-xpu-e2e-test/action.yml

```diff
@@ -110,14 +110,11 @@ runs:
                 contains "accuracy,performance" $scenario
                 $contains_status
                 if [ "${MODEL_ONLY_NAME}" == "" ];then
-                  bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 0 static 8 0 &
-                  bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 1 static 8 1 &
-                  bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 2 static 8 2 &
-                  bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 3 static 8 3 &
-                  bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 4 static 8 4 &
-                  bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 5 static 8 5 &
-                  bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 6 static 8 6 &
-                  bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 7 static 8 7 &
+                  xpu_list=($(xpu-smi discovery |awk '$2 ~ /[0-9]/ {{print $2}}'))
+                  for xpu_id in ${xpu_list[*]}
+                  do
+                    bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu ${xpu_id} static ${#xpu_list[*]} ${xpu_id} &
```

Comment:
Please double check whether it support when lost some cards on the node. For example, total 8 cards but lost card 5 in the system

Reply:
It gets cards from xpu-smi, if any card lost, it should be not in the list. As the following
![image](https://github.com/intel/torch-xpu-ops/assets/85726442/1bbd36f3-5980-470a-9b00-974498f89e24)